src/main/java/org/opentest4j/AssertionFailedError.java

```diff
@@ -40,7 +39,7 @@
 	 * and no expected/actual values.
 	 */
 	public AssertionFailedError() {
-		this((String) null);
+		this(null);
```

Comment:
The documentation for the default constructor is incorrect: the `AssertionFailedError` will have an _empty_ message. Please update accordingly.